changelog/62974.fixed

```diff
@@ -0,0 +1 @@
+Fixed test_gpg_sign unit test
```

Comment:
updates to test files do not require a changelog, only bug fixes and features. Can you remove the changelog please?

Reply:
Sure no prob. It's removed.